# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/25 22:06:47-08:00 tgraf@suug.ch 
#   [NET]: Set NLM_F_MULTI for neighbour rtnetlink messages to userspace.
#   
#   Signed-off-by: Thomas Graf <tgraf@suug.ch>
#   Signed-off-by: David S. Miller <davem@davemloft.net>
# 
# net/core/neighbour.c
#   2005/01/25 22:06:42-08:00 tgraf@suug.ch +1 -0
#   [NET]: Set NLM_F_MULTI for neighbour rtnetlink messages to userspace.
#   
#   Signed-off-by: Thomas Graf <tgraf@suug.ch>
#   Signed-off-by: David S. Miller <davem@davemloft.net>
# 
diff -Nru a/net/core/neighbour.c b/net/core/neighbour.c
--- a/net/core/neighbour.c	2005-02-03 06:30:57 -08:00
+++ b/net/core/neighbour.c	2005-02-03 06:30:57 -08:00
@@ -1469,6 +1469,7 @@
 
 	nlh = NLMSG_PUT(skb, pid, seq, event, sizeof(*ndm));
 	ndm = NLMSG_DATA(nlh);
+	nlh->nlmsg_flags = pid ? NLM_F_MULTI : 0;
 	ndm->ndm_family = n->ops->family;
 	ndm->ndm_flags = n->flags;
 	ndm->ndm_type = n->type;
